src/visitpy/visit_utils/src/engine.py

```diff
@@ -35,7 +33,37 @@ def supported_hosts():
     return res
 
 def open(**kwargs):
-    """ Launch VisIt compute engine on the current host. """
+    """ Launch VisIt compute engine on the current host.
+
+    Example usage:
+    
+    Launch engine with 36 MPI tasks using default options for this host:
+
+      engine.open(nprocs=36)
+
+    Launch engine with 36 MPI tasks using on a specific partition:
```

Comment:
typo. Either "using a" or "on a", but not both "using on".

Reply:
thanks, i pushed a fix!